cassandra/cluster.py

```diff
@@ -1076,6 +1076,43 @@ def __init__(self,
             self.status_event_refresh_window,
             schema_metadata_enabled, token_metadata_enabled)
 
+    def _create_thread_pool_executor(self, **kwargs):
+        """
+        Create a ThreadPoolExecutor for the cluster. In most cases, the built-in
+        `concurrent.futures.ThreadPoolExecutor` is used.
+
+        Python 3.7 and Eventlet cause the `concurrent.futures.ThreadPoolExecutor`
+        to hang indefinitely. In that case, the user needs to have the `futurist`
+        package so we can use the `futurist.GreenThreadPoolExecutor` class instead.
+
+        :param kwargs: All keyword args are passed to the ThreadPoolExecutor constructor.
+        :return: A ThreadPoolExecutor instance.
+        """
+        tpe_class = ThreadPoolExecutor
+        if sys.version_info[0] == 3 and sys.version_info[1] >= 7:
```

Comment:
Maybe we're expecting this to be dead code by then, but does it make sense to make this `sys.version_info[0] == 3`?

Reply:
Can you clarify this? My goal is to *only* affect versions that have the bug.

Reply:
ah, you are talking about Python 4? Didn't even think about that... but yeah maybe it's safer to just use `>= 3`.